src/backends/pyplot.jl

```diff
@@ -176,15 +176,7 @@ function add_pyfixedformatter(cbar, vals::AVec)
 end
 
 function labelfunc(scale::Symbol, backend::PyPlotBackend)
-    if scale == :log10
-        x -> PyPlot.LaTeXStrings.latexstring("10^{$x}")
-    elseif scale == :log2
-        x -> PyPlot.LaTeXStrings.latexstring("2^{$x}")
-    elseif scale == :ln
-        x -> PyPlot.LaTeXStrings.latexstring("e^{$x}")
-    else
-        x -> PyPlot.LaTeXStrings.latexstring(convert_sci_unicode(x))
-    end
+    PyPlot.LaTeXStrings.latexstring ∘ labelfunc_tex(scale)
```

Comment:
I do not really understand this `\dot` operator, bu I assume it runs latexstring once again to cover the previous else statement

Reply:
I just wanna make sure that https://github.com/JuliaPlots/Plots.jl/pull/2858 does not break because of this

Reply:
That's a `\circ` (function composition), not a `\dot`. It should behave identically to the original function. I just factored out the part that's shared between backends to `labelfunc_tex`.
I confirmed now that plots with "serif" on pyplot are also unchanged by this PR.